apps/aecore/lib/aecore/wallet/worker.ex

```diff
@@ -47,32 +48,42 @@ defmodule Aecore.Wallet.Worker do
 
   @spec get_public_key() :: binary()
   def get_public_key() do
-    get_public_key(get_aewallet_pass())
+    get_public_key("")
```

Comment:
why is the default password hardcoded now?

Reply:
No, it's not hardcoded, this is the default `derivation_path`. Now as first argument to the `get_public_key()` I have the `derivation_path`, so the full func is `get_public_key(derivation_path, passoword, network)`

Reply:
okay, can we make a variable for the default `derivation_path` then?